src/collector.ts

```diff
@@ -132,6 +161,37 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        function checkReplaceModule(dep: IDependency) {
+            let replaceDependency = replaceDependencies.get(dep.name.value);
+            if (replaceDependency === undefined) {
+                replaceDependency = replaceDependencies.get(dep.name.value + "@" + dep.version.value);
+                if(replaceDependency === undefined) {
+                    dependencies.push(dep);
+                    return;
+                }
+            }
+            dependencies.push(replaceDependency);
+        }
+
+        function checkReplacePackage(dep: IDependency, importStatement: string){
```

Comment:
This function can be merged with above function, looks like we have duplicate code and flow between this two functions.